config/sync/views.view.stanford_basic_pages.yml

```diff
@@ -572,7 +572,7 @@ display:
           relationship: none
           group_type: group
           admin_label: ''
-          default_action: ignore
+          default_action: empty
```

Comment:
I know the reasoning for this, but it also is confusing to a user that doesn't input any argument into the paragraph data field. So there's a trade off with this for the content author

Reply:
Yeah, this is another thing Ishi and I went over in a call.  It doesn't make sense to show all basic pages, and since you can use multiple terms, you can construct the list how you like.  It's a documentation thing to make sure the users know how to add the filter.

Reply:
If we get user's commenting on it, we can always adjust as necessary.